lib/deimos/activerecord/message_consumption.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
```

Comment:
The folder name should be `active_record` to be consistent with CamelCase -> snake_case conversion.